src/tools/info/ucx_info.c

```diff
@@ -100,6 +100,7 @@ int main(int argc, char **argv)
             print_opts |= PRINT_UCP_EP;
             break;
         case 't':
+            print_opts |= PRINT_DEVICES;
```

Comment:
maybe better update help message to indicate '-d' is needed for '-t' ?

Reply:
Thought about it, but for the user this way is preferred. WDYT?